.circleci/config.yml

```diff
@@ -199,6 +199,26 @@ jobs:
           command: |
             yarn test --coverage --runInBand --core
             yarn coverage
+  storybook:
+    <<: *defaults
+    steps:
+      - checkout
+      - restore_cache:
+          name: "Restore core dependencies cache"
+          keys:
+            - core-dependencies-{{ checksum "yarn.lock" }}
+      - restore_cache:
+          name: "Restore core dist cache"
+          keys:
+            - core-dist-{{ .Revision }}
+      - run:
+          name: "Link packages"
+          command: |
+            yarn install
+      - run:
+          name: "Visually test storybook"
+          command: |
+            cd examples/official-storybook; yarn chromatic
```

Comment:
Please add a root-level shortcut script for that

Reply:
Maybe it should be even made an option in `scripts/test.js`

Reply:
Is this something we want to be able to run locally?

Reply:
I believe so, at least to update the refs

Reply:
I was under the impression chromatic provided a review system to decide if a change is OK or NOT_OK ?

Reply:
If the tests are run by each dev, how does chromatic overcome the rendering differences per system?

Reply:
AFAIU they actually run in cloud even when launched locally

Reply:
> chromatic provided a review system to decide if a change is OK or NOT_OK

That's cool that they do that! And I feel convenient doing that even before submitting a PR

Reply:
Yeah, you can run it locally and it does take the snapshots in the cloud still.

The only hiccup with this is the `CHROMATIC_APP_CODE` environment variable -- it will need to be set for the test to run locally. Any thoughts on how to make that easier for devs (you can find it in the settings page at https://www.chromaticqa.com/settings?appId=5a375b97f4b14f0020b0cda3)

Reply:
@Hypnosphi -- not sure how to best add it to the `scripts/test.js` command; it seems the way that is written a command either needs to be a jest project or a `npm test` command for some subdirectory. I'm not sure we want to make it `npm test` for `examples/official-storybook` though.

Should I change the script to support a full custom command? Perhaps in a separate PR?